cron.php

```diff
@@ -115,7 +115,7 @@
 		}
 
 		$user = posix_getuid();
-		$configUser = fileowner(OC::$configDir . 'config.php');
+		$configUser = fileowner($config->getSystemValue('datadirectory', \OC::$SERVERROOT . '/data'));
```

Comment:
```suggestion
		$dataDirectoryUser = fileowner($config->getSystemValue('datadirectory', \OC::$SERVERROOT . '/data'));
```
Need to next lines too

Reply:
Maybe use getSystemValueString here?